docs/language/builtins.rst

```diff
@@ -311,6 +311,35 @@ Its underlying type is ``bytes32``, but it will be reported correctly as the ``H
 .. include:: ../examples/substrate/hash_type.sol
   :code: solidity
 
+chain_extension(uint32 ID, bytes input) returns (uint32, bytes)
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Only available on Substrate. Call the chain extension with the given ``ID`` and ``input`` data.
+Returns the return value from the chain extension and the output data.
+
+This function is a low level interface; the input and output data buffers are just ``bytes``.
```

Comment:
Total nitpick but I would replace "just" with "simply" or totally remove it. I am having a hard time explaining why it's better so I'll just leave this here as a suggestion.

Reply:
You are right. I removed the entire sentence. It's already obvious that the input and output are `bytes` by looking at the function signature. No need to repeat in words what's already obvious from the code.